tmva/pymva/test/TestRModelParserPyTorch.C

```diff
@@ -1,10 +1,11 @@
 #include <numeric>
 
-#include "PyTorchModuleModel.hxx"
-#include "PyTorchSequentialModel.hxx"
-#include "PyTorchConvolutionModel.hxx"
+#include "TMVA/RSofieReader.hxx"
+//#include "PyTorchSequentialModel.hxx"
```

Comment:
I think we can maybe remove them completely since they are no longer required here?